testing/sdk_tasks.py

```diff
@@ -184,3 +184,39 @@ def task_exec(task_name: str, cmd: str, return_stderr_in_stdout: bool = False) -
         return rc, stdout + "\n" + stderr
 
     return rc, stdout, stderr
+
+
+@retrying.retry(stop_max_attempt_number=3,
+                wait_fixed=1000,
+                retry_on_result=lambda result: not result)
+def create_text_file(task_name: str, filename: str, lines: list) -> bool:
+    output_cmd = """bash -c \"cat >{output_file} << EOL
+{content}
+EOL\"""".format(output_file=filename, content="\n".join(lines))
+    log.info("Running: %s", output_cmd)
+    rc, stdout, stderr = task_exec(task_name, output_cmd)
+
+    if rc or stderr:
+        log.error("Error creating file %s. rc=%s stdout=%s stderr=%s", filename, rc, stdout, stderr)
+        return False
+
+    linecount_cmd = "wc -l {output_file}".format(output_file=filename)
+    rc, stdout, stderr = task_exec(task_name, linecount_cmd)
```

Comment:
Has this been changed under the covers to work in 1.9 -> 1.11+?

Reply:
N/A, nothing uses this that runs on 1.9 (but still task_exec should have a shim...)

Reply:
Sorry. Was afk for a day. There are modifications to `task_exec` in #2059 that should make local executables in the sandbox work in 1.9, this would not be applicable here since `wc` is a system binary.